devices/src/virtio/mmio.rs

```diff
@@ -374,7 +375,9 @@ impl BusDevice for MmioDevice {
 
         match offset {
             0x00..=0xff if data.len() == 4 => {
-                let v = LittleEndian::read_u32(data);
+                let mut array = [0u8; 4];
+                array.copy_from_slice(data);
+                let v = u32::from_le_bytes(array);
```

Comment:
I think it would be worth having a method for this. Since to be used more than once.